bundles/org.openhab.binding.freeboxos/README.md

```diff
@@ -97,15 +97,28 @@ The *landline* thing requires the following configuration parameters:
 |------------------|-----------------|------------------------------------------------------------------------|----------|---------|
 | Refresh Interval | refreshInterval | The refresh interval in seconds which is used to poll for phone state. | No       | 2       |
 
-### Network devices: Host and WifiHost
+### Network devices: Host
 
-The *host* and *wifihost* things requires the following configuration parameters:
+The *host* things requires the following configuration parameters:
```

Comment:
```suggestion
The *host* things require the following configuration parameters:
```

Reply:
Or:
```suggestion
The *host* thing requires the following configuration parameters:
```

Reply:
Done

Reply:
Sorry, but you would have to select one of them. You did both, so now the grammar is again wrong. 😉